Snakefile

```diff
@@ -1112,6 +1124,9 @@ rule prepare_sector_network:
         network=RESDIR
         + "prenetworks/elec_s{simpl}_{clusters}_ec_l{ll}_{opts}_{sopts}_{planning_horizons}_{discountrate}_{demand}_presec.nc",
         costs="resources/" + RDIR + "costs_{planning_horizons}.csv",
+        # TODO revise the values which are currently very rough estimations
+        # &replace a temporary file with a more stable solution in future
+        cooling_costs="data/costs_cooling.csv",
```

Comment:
how could they be integrated in technology data? what can be used from there and what is missing?